translation/source/mobile.xml

```diff
@@ -34,6 +35,8 @@
   <string name="puzzleStreakAbortWarning" comment="Shown as a subtitle in the confirmation dialog for aborting puzzle streak. Prefer to not translate Puzzle Streak">You will lose your current streak and your score will be saved.</string>
   <string name="puzzleThemesSubtitle" comment="Shown as a description for the 'Puzzle themes' mode">Play puzzles from your favorite openings, or choose a theme.</string>
   <string name="puzzlesTab" comment="The tab in bottom navigation bar. Must be very short (10 characters max) to avoid ugly text shifting issues." maxLength="10">Puzzles</string>
+  <string name="ratedGame" comment="This is shown in quick actions to show the name is rated">(Rated)</string>
+  <string name="recentlyPlayed" comment="Subtitle shown in quick actions for a recently played game">Recently played</string>
```

Comment:
There is already the `arenaRecentlyPlayed` that we can use, so please remove that one.

We should be conservative when adding new keys. For new features it is even recommended to hard code the string at first. Will update the internationalisation doc to make it more clear.